### PR TITLE
Fix atol for test_preloaded_multi_sgd

### PR DESCRIPTION
**Pull request by Caenorst**
## Description ##
Fix atol for test_preloaded_multi_sgd
Following Issue: #16345 
Fixes #16122 

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- [x] Code is well-documented: 

### Changes ###
- Change atol on fp16 from 1e-3 to 1e-2

## Comments ##
- As asked on the issue thread, tested 10k times on my machine

### CODE REVIEW

File: tests/python/gpu/test_operator_gpu.py [hunk 1]
```diff
@@ -396,7 +396,7 @@ def _assert_all_almost_equal(lhs_list, rhs_list, rtol, atol):
             assert_almost_equal(lhs.asnumpy(), rhs.asnumpy(), rtol=rtol, atol=atol)
     if dtype == 'float16':
         rtol = 1e-3
-        atol = 1e-3
+        atol = 1e-2
     else:
         rtol = 1e-5
         atol = 1e-6
```
Comment (ChaiBapchya) [inline tests/python/gpu/test_operator_gpu.py]: what about when its !float16?

Comment (Caenorst) [reply tests/python/gpu/test_operator_gpu.py]: I'm testing 10k including non fp16, so I guess the other tolerances are fine

**Comment by Caenorst** [comment]
> Thanks for the quick fix. @Caenorst
> Can you add "Fixes #PR" that would auto close the issue if/when this PR gets merged.
> 
> Thanks again for testing it 10k times before the PR!

Where should I add this "Fixes #16122 " ?

**Comment by ChaiBapchya** [comment]
Generally in the description of the PR

**Pull request merged**